platform/lib/schema/index.ts

```diff
@@ -483,6 +542,14 @@ export function arrayOf<T>(
   return new ArraySetting(itemSetting, options);
 }
 
+export function mapOf<K, V>(
```

Comment:
__nit:__ it would suuuuuuuper-awesome if you can add proper `JSDoc` for this exported function (with tiny example maybe) :)